packages/muelu/research/regionMG/example/CMakeLists.txt

```diff
@@ -0,0 +1,46 @@
+INCLUDE_DIRECTORIES(${CMAKE_CURRENT_SOURCE_DIR})
+INCLUDE_DIRECTORIES(${CMAKE_CURRENT_SOURCE_DIR}/../../../test/unit_tests)
+INCLUDE_DIRECTORIES(${CMAKE_CURRENT_SOURCE_DIR}/../src)
+
+# This test requires Tpetra and Amesos2, so it's only included if Tpetra is enabled
+IF (${PACKAGE_NAME}_ENABLE_Tpetra AND ${PACKAGE_NAME}_ENABLE_Amesos2)
+
+
+  TRIBITS_ADD_EXECUTABLE(
+    RegionExampleDriver
+    SOURCES ../test/structured/Driver_Structured_Regions.cpp # Reuse the source code form the test directory
+    COMM serial mpi
+    )
+
+  TRIBITS_COPY_FILES_TO_BINARY_DIR(StructuredRegion_example_cp
+    SOURCE_FILES
+      elasticity_3d.xml
+      poisson_3d.xml
+      structured_unstructured_1dof.xml
+    )
+
+  TRIBITS_ADD_TEST(
+    RegionExampleDriver
+    NAME "Structured_Region_Example_Elasticity3D"
+    ARGS "--linAlgebra=Tpetra --xml=elasticity_3d.xml --matrixType=Elasticity3D --nx=28 --ny=28 --smootherIts=2"
+    COMM serial mpi
+    NUM_MPI_PROCS 4
+    )
+
+  TRIBITS_ADD_TEST(
+    RegionExampleDriver
+    NAME "Structured_Region_Example_Poisson3D_27pt_stencil"
+    ARGS "--linAlgebra=Tpetra --xml=poisson_3d.xml --matrixType=Brick3D --nx=28 --ny=28 --smootherIts=2"
```

Comment:
Do we want to specify `--nz` as well?

Reply:
@pohm01 Good point! Since this is an example, we should definitely specify that to be more clear about the setup. I already pushed the necessary change.